.github/workflows/tests.yml

```diff
@@ -25,15 +30,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11-dev"]
```

Comment:
According to [this comment](https://github.com/actions/setup-python/issues/213#issuecomment-1191293053), this should be:

```suggestion
        python-version: ["3.6-dev", "3.7-dev", "3.8-dev", "3.9-dev", "3.10-dev", "3.11-dev"]
```

Reply:
On second thought, better to add `"3.11"` here and add `-dev` to the setup-python step below.

Reply:
```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```

Reply:
Yes, nice!